js/forum/dist/extension.js

```diff
@@ -62,6 +63,27 @@ System.register("wuethrich44/sso/main", ["flarum/extend", "flarum/app", "flarum/
                     items.remove('changeEmail');
                     items.remove('changePassword');
                 }
+
+                function checkRemoveAccountSection(items) {
+                    if (items.has('account')) {
```

Comment:
This looks a little bit too complicated to me. Either remove the complete account section or check if there are any children and remove them like this:
```javascript
if (items.get('account').props.children.length === 0) {
    items.remove('account')
}
```